tests/rptest/clients/rpk_remote.py

```diff
@@ -34,9 +34,13 @@ def config_set(self, key, value, format=None, path=None, timeout=30):
 
     def debug_bundle(self, working_dir):
```

Comment:
`working_dir` should probably be changed to `output_file`, to match the new usage